modules/microshift-manifests-removal-scenario-rpm.adoc

```diff
@@ -15,16 +15,16 @@ Use the following procedure in the data removal scenario for RPM systems to comp
 +
 [source,terminal]
 ----
-$ sudo mkdir -p _<path_of_delete_directory>_ <1>
+$ sudo mkdir -p <path_of_delete_directory> <1>
 ----
-<1> Replace `_<path_of_delete_directory>_` with the path of the delete subdirectory, for example, `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
+<1> Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
 . Move the manifest file into one of the `delete` subdirectories under the configured manifests directory by running the following command:
 +
 [source,terminal]
 ----
-$ [sudo] mv _<path_of_manifests>_ _<path_of_delete_directory>_ <1>
+$ [sudo] mv <path_of_manifests> <path_of_delete_directory> <1>
 ----
-<1> Replace `_<path_of_manifests>_` with the path of the manifest to be deleted, for example, `/etc/microshift/manifests.d/010-SOME-MANIFEST`. Replace `_<path_of_delete_directory>_` with the path of the delete subdirectory, for example, `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
+<1> Replace `_<path_of_manifests>_` with the path of the manifest to be deleted, for example, `/etc/microshift/manifests.d/010-SOME-MANIFEST`. Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
```

Comment:
Out of scope suggestion for your consideration, up to you if you're interested. But in cases like this, where you are trying to use 1 callout to talk about multiple user-replaced values on the same line, it can get a little clunky and confusing. 

For these situations, we're allowed to use [description lists](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks). Here's what that might look like here:
```suggestion
+
where:

`_<path_of_manifests>_`:: Specifies the path of the manifest to be deleted, for example `/etc/microshift/manifests.d/010-SOME-MANIFEST`. 
`_<path_of_delete_directory>_`:: Specifies one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
```

If you do choose to go with this, you must also delete the callout on line 25

Reply:
Thank you for the suggestion.  done